_posts/2019-06-20-load-image-data.md

```diff
@@ -206,7 +206,8 @@ To load a single image, we use [Scikit-Image](https://scikit-image.org/):
 597
 
 >>> import skimage.io
```

Comment:
Guessing we intend to drop this as well.

```suggestion
```

Reply:
Since we still it below in `skimage.io.imshow`, I will move it there.

Reply:
Good point. Thanks!